hack/upgrade-e2e-test.sh

```diff
@@ -20,9 +20,11 @@
 go run "$(dirname $0)/e2e.go" -v -build -up -version="v0.14.0" -test -check_version_skew=false
 if [ $? -eq 0 ]; then
 	echo "Tests on initial version succeeded. Proceeding with push and second set of tests."
-	go run "$(dirname $0)/e2e.go" -v -push -version="" -test -check_version_skew=false
+	go run "$(dirname $0)/e2e.go" -v --test --test_args='--ginkgo.focus=Skipped.*Cluster.*upgrade' -check_version_skew=false
 else
 	echo "Tests on initial version failed. Skipping tests on second version."
 fi
+go run "$(dirname $0)/e2e.go" -v -build -up -version="" -test -check_version_skew=false
```

Comment:
Can you please add an `echo` here that says the second set of tests are being run now?